### PR TITLE
Fix link to binary-installation method in README.md

### LINKED ISSUE #902: No proper link to Binary Installation method in README.md
Link for the **Binary (Prefered method)** method in the `README.md` opens https://github.com/kubernetes/kompose/blob/master/README.md instead of jumping to the proper section from the file.

### PR DESCRIPTION
**Pull request by bhavin192**
Now it links to the section of same file.
Also fixes typo in word preferred.
fixes #902

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -24,12 +24,12 @@ Other examples are provided in the _examples_ [directory](./examples).
...
 ## Installation
 
-We have multiple ways to install Kompose. Our prefered method is downloading the binary from the latest GitHub release.
+We have multiple ways to install Kompose. Our preferred method is downloading the binary from the latest GitHub release.
 
 Our entire list of installation methods are located in our [installation.md](/docs/installation.md) document.
 
 Installation methods:
-  - [Binary (Prefered method)](README.md)
+  - [Binary (Preferred method)](#binary-installation)
```
Comment (cdrage) [inline README.md]: I think this should be `/docs/installation.md#binary-installation` otherwise lgtm.

**Comment by surajnarwade** [comment]
Thanks @bhavin192 for your contribution, after changes suggested by @cdrage , good to go :)

File: README.md (hunk 1 above)
Comment (bhavin192) [reply README.md]: We don't have a section named `binary-installation` in `/docs/installation.md` (actual method is there though)
but the same installation method is present in `README.md` itself, so I thought linking to the section from `README.md` will be better solution.
should I link to `/docs/installation.md` instead?

Comment (cdrage) [reply README.md]: @bhavin192 sorry, link to: https://github.com/kubernetes/kompose/blob/master/docs/installation.md#github-release

so: `docs/installation.md#github-release`

Comment (bhavin192) [reply README.md]: @cdrage I got it now, so the section is like a index of `installation.md` but again won't it be little bit confusing for user? as we have instructions there on the `README.md` itself (for binary installation) and we are redirecting to `installation.md` for the same instructions?
Can we have list of all methods after the Binary Installation method (`README.md`) ?

**Comment by cdrage** [comment]
IMO. Having it separate would be best as it shows other methods for
installation too.

**Comment by cdrage** [comment]
LGTM! Thanks!

**Pull request merged**